package.json

```
@@ -30,7 +30,7 @@
   },
   "devDependencies": {
     "async": "2.6.0",
-    "chromedriver": "^75.0",
+    "chromedriver": "^80.0.0",
```

```suggestion
    "chromedriver": "80.0.1",
```

Node v8 support was dropped on `80.0.2`:  giggio/node-chromedriver#261

---

👍